src/components/Output/index.tsx

```diff
@@ -2,6 +2,7 @@ import { h, Component } from 'preact';
 import PinchZoom from './custom-els/PinchZoom';
 import './custom-els/PinchZoom';
 import './custom-els/TwoUp';
+import './custom-els/RangeInput';
```

Comment:
Would you prefer it be imported from the options that require range inputs? I didn't want to since we arbitrarily nested these CE's within directories specific to each component, seemed like reaching across a boundary.